Doc/library/xmlrpc.server.rst

```diff
@@ -185,6 +192,37 @@ server::
    # Print list of available methods
    print(s.system.listMethods())
 
+Since version 3.7, :meth:`register_function` can also be used as a decorator. The
+previous server example can register functions in a decorator way::
+
+   from xmlrpc.server import SimpleXMLRPCServer
+   from xmlrpc.server import SimpleXMLRPCRequestHandler
+
+   class RequestHandler(SimpleXMLRPCRequestHandler):
+       rpc_paths = ('/RPC2',)
+
+   with SimpleXMLRPCServer(("localhost", 8000),
+                           requestHandler=RequestHandler) as server:
+       server.register_introspection_functions()
+
+       # Register pow() function; this will use the value of
+       # pow.__name__ as the name, which is just 'pow'.
+       server.register_function(pow)
+
+       # Register a function under a different name, using
+       # register_function as a decorator. *name* can only be given
+       # as a keyword argument.
```

Comment:
But the name parameter is not keyword-only. It is positional-or-keyword.

The former wording looked correct to me, the changed wording don't.